apps/root/src/common/components/transaction-steps/index.tsx

```diff
@@ -346,32 +420,41 @@ const buildApproveTokenItem = ({
               )}
             </ContainerBox>
           ) : (
-            <Button
-              variant="contained"
-              color="secondary"
-              fullWidth
-              size="large"
-              disabled={isPendingTransaction}
-              onClick={() => (receipt && !isPendingTransaction ? setShowReceipt(true) : onGoToEtherscan(hash))}
-            >
-              {isPendingTransaction ? (
-                <CircularProgress size={20} />
-              ) : receipt ? (
-                <FormattedMessage description="viewReceipt" defaultMessage="View receipt" />
-              ) : (
-                <FormattedMessage description="viewExplorer" defaultMessage="View in explorer" />
+            <ContainerBox gap={4} alignItems="center">
+              <Button
+                variant="contained"
+                color="secondary"
+                fullWidth
+                size="large"
+                disabled={isPendingTransaction}
+                onClick={() => (receipt && !isPendingTransaction ? setShowReceipt(true) : onGoToEtherscan(hash))}
+              >
+                {isPendingTransaction ? (
+                  <CircularProgress size={20} />
+                ) : receipt ? (
+                  <FormattedMessage description="viewReceipt" defaultMessage="View receipt" />
+                ) : (
+                  <FormattedMessage description="viewExplorer" defaultMessage="View in explorer" />
+                )}
+              </Button>
+              {!isPendingTransaction && receipt && (
+                <ContainerBox gap={2} alignItems="center">
+                  <StyledTickCircleIcon />
+                  <Typography variant="body" fontWeight="600">
+                    <FormattedMessage
+                      description="approveSellAmount"
+                      defaultMessage="Sell {amount} {symbol}"
```

Comment:
Is this correct? Isn't this the approval step?

Reply:
I found this label in the design, and it seemed prudent to me to add it, even though it was on another step. I should have informed you beforehand 😅 
![Screenshot 2024-02-08 at 11 20 11](https://github.com/Mean-Finance/dca-fe/assets/144699868/bf186c49-3ea1-4dae-92c8-8583295e0562)

PS: After reviewing it, I think it should be "Approve 20 NEXT"

Reply:
Yup it should be approve